src/plugins/home/public/application/components/home.js

```diff
@@ -163,6 +163,7 @@ export class Home extends Component {
               addBasePath={addBasePath}
               solutions={solutions}
               directories={directories}
+              branding={getServices().injectedMetadata.getBranding()}
```

Comment:
injectedMetadata will always be set right?

Reply:
Yes. injectedMetadata.getBranding() is always set.